sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/UnitTestResultsImportSensor.java

```diff
@@ -76,37 +76,40 @@ public void execute(SensorContext context) {
   }
 
   void analyze(SensorContext context, UnitTestResults unitTestResults) {
-    UnitTestResults aggregatedResults = unitTestResultsAggregator.aggregate(wildcardPatternFileProvider, unitTestResults);
+    try {
+      UnitTestResults aggregatedResults = unitTestResultsAggregator.aggregate(wildcardPatternFileProvider, unitTestResults);
```

Comment:
To avoid having a big block of code inside a try-catch statement, I'd suggest to keep the previous method as a helper ex: `saveTestMetrics` method, and just wrap that call inside the try-catch